Storage/Storage/Model/MetaData+CoreDataProperties.swift

```diff
@@ -12,6 +12,7 @@ extension MetaData {
     @NSManaged public var value: String?
     @NSManaged public var metadataID: Int64
     @NSManaged public var order: Order?
+    @NSManaged public var product: Product?
```

Comment:
I think we should the comment in [this networking file](https://github.com/woocommerce/woocommerce-ios/blob/trunk/Networking/Networking/Model/MetaData.swift#L4-L6).

Reply:
Good catch, done 508d94affe85bc22d1c96059ab34b5d6b9aa9b1c